README.rst

```diff
@@ -141,6 +141,15 @@ Please note the following guidelines for contributing:
   * If you are adding new functionality, you must include basic tests
     and documentation.
 
+
```

Comment:
Since I did not know about this feature I'd like to add this line to the docs:

> Running the tests
> =================
> Install the dependencies specified in requirements.txt and run::
> 
>     setup.py test
>